### DISCUSSION TITLE
Backup jobs won't launch from the UI due to JavaScript error

### DISCUSSION THREAD

**Discussion by k8n** [spatie/nova-backup-tool#145] [General]
Nova 4.35.

Apparently `delayedHideMenu()` is not a valid method on the Dropdown component.

```
runtime-core.esm-bundler.js:259 TypeError: this.$refs.backupDropdownMenu.delayedHideMenu is not a function
    at Proxy.createPartialBackup (BackupTool:1:21405)
    at BackupTool:1:23070
...
```

https://github.com/spatie/nova-backup-tool/blob/251d906d24054644ffd834057705d375f20211b0/resources/js/pages/BackupTool.vue#L142

Do I start a PR, or could someone squash this one for us all?

**Comment by scramatte** [comment]
Same issue here
I've already fork  the repository to apply all fixes necessary to get it running with  laravel-backup v9
It looks that the original package repository is not maintained a lot.

https://github.com/operativeit/spatie-nova-backup-tool

**Reply by madebycaliper** [reply]
Thank you @scramatte!! is there a way I can install this via composer? the instructions still show the Spatie `composer require` statement and I can't find this on packagist.

**Reply by madebycaliper** [reply]
There was a problem with your `composer.json` file – the name and homepage attributes weren't updated to match the repo slug. I forked and fixed on my end but it might be helpful for you to update that in your repo as well 🙏
